Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM python:3.8.5-slim@sha256:9c84459051ee7e9d386b24ee5468352d52b6789a5d4a8cf6a649a8a1c6ad5636
+
+# https://github.com/mozilla-services/Dockerflow/blob/master/docs/building-container.md
+ARG USER_ID="10001"
+ARG GROUP_ID="10001"
+
+ENV HOME=/app
+RUN groupadd --gid ${GROUP_ID} app && \
+    useradd --uid ${USER_ID} --gid ${GROUP_ID} --home-dir /app --create-home app
+
+RUN apt-get update && \
+    apt-get install -y build-essential 
+
```

Comment:
Borrowing from https://github.com/mozilla/missioncontrol-v2/blob/master/Dockerfile:

```suggestion
# clean up apt-get cruft
RUN apt-get autoremove -y && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Reply:
I'll add that. It knocks 18mb off the size of the image.